<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatherPad::SessionDialog</class>
 <widget class="QDialog" name="sessionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Session Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="mainContainer" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;b&gt;Save/Restore Session&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QListWidget" name="listWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="openBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Open the selected session(s).</string>
        </property>
        <property name="text">
         <string>&amp;Open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="removeBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Remove the selected session(s).</string>
        </property>
        <property name="text">
         <string>&amp;Remove</string>
        </property>
        <property name="icon">
         <iconset theme="edit-clear"/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="clearBtn">
        <property name="text">
         <string>Remove &amp;All</string>
        </property>
        <property name="icon">
         <iconset theme="edit-delete"/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="saveBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Save the curent session under the given title.</string>
        </property>
        <property name="text">
         <string>&amp;Save</string>
        </property>
        <property name="icon">
         <iconset theme="document-save"/>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="LineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Type a name to save session</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="promptContainer" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="warningLabel">
        <property name="text">
         <string>&lt;b&gt;Warning&lt;/b&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="promptLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="confirmBtn">
        <property name="text">
         <string>O&amp;K</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignRight">
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Canc&amp;el</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" alignment="Qt::AlignRight">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
